plugins/tekton/src/components/common/Status.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import {
+  BanIcon,
+  ClipboardListIcon,
+  HourglassHalfIcon,
+  HourglassStartIcon,
+  NotStartedIcon,
+  SyncAltIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+import StatusIconAndText from './StatusIconAndText';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from './icons';
+
+export type StatusComponentProps = {
+  title?: string;
+  iconOnly?: boolean;
+  noTooltip?: boolean;
+  className?: string;
+  popoverTitle?: string;
+};
+
+export type StatusProps = StatusComponentProps & {
+  status: string;
+  children?: React.ReactNode;
+};
+
+export const DASH = '-';
```

Comment:
why export? and in this file it's used only once so can be used directly as well or move to const